src/pal/src/include/pal/context.h

```diff
@@ -468,6 +472,19 @@ inline static void CONTEXTSetPC(LPCONTEXT pContext, DWORD64 pc)
 #endif
 }
 
+inline static DWORD64 CONTEXTGetFP(LPCONTEXT pContext)
```

Comment:
Up to you, but I think 'CONTEXTGetFP' could be a little confusing since we use FP to refer to floating-point stuff in a lot of our context-handling code.